src/cljs/rems/application.cljs

```diff
@@ -735,40 +735,44 @@
                                 :readonly readonly?
                                 :approved (get licenses (:id license)))]))])]}]))
 
+
+;; FIXME Why do we have both this and dynamic-event->event?
```

Comment:
`dynamic-event->event` converts dynamic events to the legacy event format used by the round-based workflow. After removing the round-based workflow we should be able to clean up this.